backend-api/src/main/resources/application.conf

```diff
@@ -0,0 +1,12 @@
+server.port=8080
+
+cassandra {
+    contactpoints=localhost
+    port=9042
+    keyspace=pet_mgt
+}
+
+featureFlags {
+    activate-cassandra-rw=false
+    dummy-toggle=true
+}
```

Comment:
add spaces before and after `=` to make it easier to read these:

```
server.port=8080

cassandra {
    contactpoints = localhost
    port = 9042
    keyspace = pet_mgt
}

featureFlags {
    activate-cassandra-rw = false
    dummy-toggle = true
} 
```

Reply:
kk, I'll do it. Just followed the examples in the documentation.